src/terminal/adapter/adaptDispatchGraphics.cpp

```diff
@@ -333,213 +43,227 @@ static constexpr bool _isDefaultColorOption(const DispatchTypes::GraphicsOptions
 //      Xterm index will use the param that follows to use a color from the preset 256 color xterm color table.
 // Arguments:
 // - options - An array of options that will be used to generate the RGB color
-// - rgbColor - Location to place the generated RGB color into.
-// - isForeground - Location to place whether or not the parsed color is for the foreground or not.
-// - optionsConsumed - Location to place the number of options we consumed parsing this option.
+// - attr - The attribute that will be updated with the parsed color.
+// - isForeground - Whether or not the parsed color is for the foreground or not.

```

Comment:
```suggestion
// - isForeground - Whether or not the parsed color is for the foreground.
```

Reply:
I don't trust github not to screw up the line endings if I accept this here, but I'll make another commit with your suggested comment changes.